### ISSUE TITLE
Not created new connection

### ISSUE THREAD

**Issue by rugk** [threema-ch/threema-web#434]
## Expected Behavior

Work.

## Current Behavior

Did not hide QR code.

**Browser log:**
<details>

```
22:22:42.622 [StateService] Reset
angular.js:14199:18
22:22:42.639 [WebClientService] Reset initialization steps
angular.js:14199:18
22:22:42.773 Detected browser: Firefox 58
angular.js:14199:18
22:22:42.773 [WebClientService] Reset initialization steps
angular.js:14199:18
22:22:42.773 [WelcomeController] Initialize session by scanning QR code...
angular.js:14199:18
22:22:42.774 [StateService] Reset
angular.js:14199:18
22:22:42.787 [SaltyRTC.KeyStore] New public key: afe06ac3460ecbc31b0cdfc22993f54b1ae524db79d4a8e83f1770d072c7747e
saltyrtc-client.es5.js:467:13
22:22:42.789 [SaltyRTC.AuthToken] Generated auth token
saltyrtc-client.es5.js:526:13
22:22:42.789 [WelcomeController] Checking if the session is already open in another tab or window
angular.js:14199:18
22:22:42.790 Starting WebClientService...
angular.js:14199:18
22:22:42.790 [SaltyRTC.Client] New event: state-change
saltyrtc-client.es5.js:2934:13
22:22:42.790 [SaltyRTC.Client] New event: state-change:new
saltyrtc-client.es5.js:2934:13
22:22:42.791 [SaltyRTC.Client] New event: state-change
saltyrtc-client.es5.js:2934:13
22:22:42.791 [SaltyRTC.Client] New event: state-change:ws-connecting
saltyrtc-client.es5.js:2934:13
22:22:42.791 [SaltyRTC.Initiator] Opening WebSocket connection to wss://saltyrtc-af.threema.ch:443/afe06ac3460ecbc31b0cdfc22993f54b1ae524db79d4a8e83f1770d072c7747e
saltyrtc-client.es5.js:1172:13
22:22:42.857 [StateService] Signaling connection state: new => new
angular.js:14199:18
22:22:42.916 [StateService] Signaling connection state: new => ws-connecting
angular.js:14199:18
22:22:42.919 [VersionService] Using Threema Web version 1.7.0-gh
angular.js:14199:18
22:23:42.832 Firefox kann keine Verbindung zu dem Server unter wss://saltyrtc-af.threema.ch/afe06ac3460ecbc31b0cdfc22993f54b1ae524db79d4a8e83f1770d072c7747e aufbauen.
saltyrtc-client.es5.js:1165:22
22:23:42.835
[SaltyRTC.Initiator] General WebSocket error 
error
bubbles: false
cancelBubble: false
cancelable: false
composed: false
currentTarget: null
defaultPrevented: false
eventPhase: 0
explicitOriginalTarget: WebSocket { url: "wss://saltyrtc-af.threema.ch/afe06ac3460ecbc31b0cdfc22993f54b1ae524db79d4a8e83f1770d072c7747e", readyState: 3, bufferedAmount: 0, … }
isTrusted: true
originalTarget: WebSocket { url: "wss://saltyrtc-af.threema.ch/afe06ac3460ecbc31b0cdfc22993f54b1ae524db79d4a8e83f1770d072c7747e", readyState: 3, bufferedAmount: 0, … }
target: WebSocket { url: "wss://saltyrtc-af.threema.ch/afe06ac3460ecbc31b0cdfc22993f54b1ae524db79d4a8e83f1770d072c7747e", readyState: 3, bufferedAmount: 0, … }
timeStamp: 60886.528323
type: "error"
__proto__: EventPrototype { stopPropagation: stopPropagation(), stopImmediatePropagation: stopImmediatePropagation(), preventDefault: preventDefault(), … }
saltyrtc-client.es5.js:988:13
22:23:42.841 [SaltyRTC.Client] New event: connection-error
saltyrtc-client.es5.js:2934:13
22:23:42.842
Connection error: 
Object { type: "connection-error" }
angular.js:14199:18
consoleLog/<
https://[…]/node_modules/angular/angular.js:14199:18
init/<
https://[…]/dist/app.js:15921:17
callHandler
https://[…]/node_modules/@saltyrtc/client/dist/saltyrtc-client.es5.js:2976:28
emit
https://[…]/node_modules/@saltyrtc/client/dist/saltyrtc-client.es5.js:2945:25
Signaling/this.onError
https://[…]/node_modules/@saltyrtc/client/dist/saltyrtc-client.es5.js:989:13
22:23:42.844 [SaltyRTC.Initiator] Closed WebSocket connection
saltyrtc-client.es5.js:995:17
22:23:42.845 [SaltyRTC.Client] New event: state-change
saltyrtc-client.es5.js:2934:13
22:23:42.845 [SaltyRTC.Client] New event: state-change:closed
saltyrtc-client.es5.js:2934:13
22:23:42.845 [SaltyRTC.Client] New event: connection-closed
saltyrtc-client.es5.js:2934:13
22:23:42.846 Connection closed: 
{…}
data: 1006
type: "connection-closed"
__proto__: Object { … }
angular.js:14199:18
22:23:42.847 [StateService] Connection buildup state: connecting => closed
angular.js:14199:18
22:23:42.848 [StateService] Signaling connection state: ws-connecting => closed
```
</details>

**App log:**
<details>

```
Wed Feb 14 22:23:14 GMT+01:00 2018	releaseConnectionLinger: source = activityPaused, timeout = 60000
Wed Feb 14 22:23:14 GMT+01:00 2018	Cancel alarm type 1
Wed Feb 14 22:23:14 GMT+01:00 2018	Schedule alarm type 1 in 59993ms
Wed Feb 14 22:23:14 GMT+01:00 2018	acquireConnection: source = activityResumed, refCount = 1
Wed Feb 14 22:23:14 GMT+01:00 2018	another connection is already active
Wed Feb 14 22:23:14 GMT+01:00 2018	releaseConnectionLinger: source = activityPaused, timeout = 60000
Wed Feb 14 22:23:14 GMT+01:00 2018	Cancel alarm type 1
Wed Feb 14 22:23:14 GMT+01:00 2018	Schedule alarm type 1 in 59992ms
Wed Feb 14 22:23:14 GMT+01:00 2018	acquireConnection: source = activityResumed, refCount = 1
Wed Feb 14 22:23:14 GMT+01:00 2018	another connection is already active
Wed Feb 14 22:23:17 GMT+01:00 2018	releaseConnectionLinger: source = activityPaused, timeout = 60000
Wed Feb 14 22:23:17 GMT+01:00 2018	Cancel alarm type 1
Wed Feb 14 22:23:17 GMT+01:00 2018	Schedule alarm type 1 in 59995ms
Wed Feb 14 22:23:17 GMT+01:00 2018	acquireConnection: source = activityResumed, refCount = 1
Wed Feb 14 22:23:17 GMT+01:00 2018	another connection is already active
Wed Feb 14 22:23:17 GMT+01:00 2018	releaseConnectionLinger: source = activityPaused, timeout = 60000
Wed Feb 14 22:23:17 GMT+01:00 2018	Cancel alarm type 1
Wed Feb 14 22:23:17 GMT+01:00 2018	Schedule alarm type 1 in 59994ms
Wed Feb 14 22:23:18 GMT+01:00 2018	acquireConnection: source = activityResumed, refCount = 1
Wed Feb 14 22:23:18 GMT+01:00 2018	another connection is already active
Wed Feb 14 22:23:18 GMT+01:00 2018	releaseConnectionLinger: source = activityPaused, timeout = 60000
Wed Feb 14 22:23:18 GMT+01:00 2018	Cancel alarm type 1
Wed Feb 14 22:23:18 GMT+01:00 2018	Schedule alarm type 1 in 59993ms
Wed Feb 14 22:23:22 GMT+01:00 2018	SessionInstanceService[5]: Initialize SessionInstanceServiceImpl
Wed Feb 14 22:23:22 GMT+01:00 2018	Webclient Session Wakeup: getInstance
Wed Feb 14 22:23:22 GMT+01:00 2018	SessionInstanceService[5]: Starting Threema Web session
Wed Feb 14 22:23:22 GMT+01:00 2018	SessionStateManager[5]: Session state changed from NEW to CONNECTING
Wed Feb 14 22:23:22 GMT+01:00 2018	acquireConnection: source = activityResumed, refCount = 1
Wed Feb 14 22:23:22 GMT+01:00 2018	another connection is already active
Wed Feb 14 22:23:32 GMT+01:00 2018	SessionStateManager[5]: Session state changed from CONNECTING to ERROR
Wed Feb 14 22:23:32 GMT+01:00 2018	SessionStateManager[5]: Session state changed from ERROR to NEW
Wed Feb 14 22:24:18 GMT+01:00 2018	Alarm type 1 received
Wed Feb 14 22:24:18 GMT+01:00 2018	Handling alarm type 1
Wed Feb 14 22:24:18 GMT+01:00 2018	cleanupConnection: refCount = 1 - not cleaning up
Wed Feb 14 22:24:22 GMT+01:00 2018	releaseConnectionLinger: source = activityPaused, timeout = 60000
Wed Feb 14 22:24:22 GMT+01:00 2018	Cancel alarm type 1
Wed Feb 14 22:24:22 GMT+01:00 2018	Schedule alarm type 1 in 59938ms
Wed Feb 14 22:25:22 GMT+01:00 2018	Alarm type 1 received
Wed Feb 14 22:25:22 GMT+01:00 2018	Handling alarm type 1
Wed Feb 14 22:25:22 GMT+01:00 2018	stopConnection
Wed Feb 14 22:25:22 GMT+01:00 2018	ThreemaConnection state changed: DISCONNECTED Port: 5222 IPv6: false
Wed Feb 14 22:25:22 GMT+01:00 2018	cleanupConnection: connection closed
```
</details>

## Steps to Reproduce (for bugs)
(fresh session with password)

1. For some reason I could nearly finish a connection creation process although a red bar was shown in the Threema app (devices are in same WLAN), but at a high number (80% or so) it stopped and "connection closed".
2. So I reloaded.
3. Swipe Threema app away (i.e. "close" it) and reopen. -> red bar gone
4. Try to restore.
5. Did not work -> (AFAIK without reloading!) I've enabled debug logging and tried again

I suspect it somehow failed the timing or so when I've had two connections open to the signalling server.

## Your Environment

* Threema Web version: 1.7.0-gh
* Threema for Android version: beta
* Threema for Android source: Play Store
* Browser name and version: Firefox 58
* Operating system and version: Linux

**Comment by lgrahl** [comment]
Can you rephrase this report? I don't understand it...

**Comment by rugk** [comment]
Sorry, but >1 year later I also really have no idea what I wrote. It seems to be some awkward behaviour when I created a persistent session while the Threema _app_ had no connection (red bar)  or whatever that is…
